src/style.css

```diff
@@ -1 +1,110 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color: #3d4e61;
+  --border-color: #e1e7ed;
+  --main-bs-color: rgba(61, 78, 97, 0.1);
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, Arials, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+/* #region larger search bar & identical search bars code */
+
+.search-bar {
+  display: flex;
+  align-items: center;
+
+  box-shadow: 0 1px 8px var(--main-bs-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  color: var(--color);
+}
+
+    .search-bar--larger {
+      margin: 20px 8px;
+      min-height: 70px;
+    }
+
+  .icon {
+    background-image: url(./images/Search.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+  }
+
+    .icon--search {
+      width: 19px;
+      height: 19px;
+      margin: 0 17px 0 26px;
+    }
+
+    .input {
+    font-family: inherit;
+    font-weight: 300;
+    width: 100%;
+    height: 100%;
+    outline: none;
+    margin-right: 10px;
+    border: 0;
+    background: none;
+  }
+
+    .input--larger {
+      font-size: 16px;
+    }
+
+    .input::placeholder {
+      color: inherit;
+    }
+
+/* #endregion */
+
+/* #region smaller search bar */
+
+.search-bar--mini {
+  margin-inline: 8px ;
+  min-height: 42px;
+}
+
+  .icon--search-mini {
+    width: 11px;
+    height: 11px;
+    margin: 0 9px 0 13px;
+  }
+
+    .input--mini {
+      font-size: 14px;
+    }
+
+/* #endregion */
+
+.search-bar:hover {
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-bar:focus-within {
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.input:focus {
+  background: none;
+  font-weight: 900;
```

Comment:
tru do not set font-weight for focus. looks like such font styles should be for input, and for placeholder set font styles that in design. it can help fix tests.

Reply:
the bugs are fixed and the test is now working